frontend/packages/client/src/components/Community/CommunityCard.js

```diff
@@ -101,9 +72,16 @@ const CommunityCard = ({
   }
 
   return (
-    <Link to={`/community/${id}?tab=about`} style={{ color: "inherit" }}>
-      {Body}
-    </Link>
+    <>
+      <Link to={`/community/${id}?tab=about`} style={{ color: "inherit" }}>
+        {Body}
+      </Link>
+      <div
+        style={{ position: "absolute", margin: 0, top: "50px", right: "50px" }}
+      >
```

Comment:
setting this one as absolute causes overlapping with long text communities:
maybe added as default and align it with bulma columns ?

<img width="503" alt="Screen Shot 2022-05-23 at 12 04 54" src="https://user-images.githubusercontent.com/7526740/169850358-fba69675-4af8-4f79-83a0-1e2214bf30be.png">